src/main/kotlin/org/koitharu/kotatsu/parsers/model/Manga.kt

```diff
@@ -52,7 +52,12 @@ public data class Manga private constructor(
 	/**
 	 * Author of the manga, may be null
 	 */
+	@Deprecated("Please use authors")
 	@JvmField public val author: String?,
+	/**
+	 * Authors of the manga
+	 */
+	@JvmField public val authors: Set<MangaTag>,
```

Comment:
Not sure about providing authors as `MangaTag`, `Set<String>` would be better for compatibility